_data/beps_completed.yml

```diff
@@ -39,27 +48,39 @@
     - name: Robert Oostenveld
   content:
     - raw
+  google_doc_created: 2017-06
+  pull_request_created: 2018-12
+  pull_request_merged: 2019-03
 
 - number: "007"
   title: Hierarchical Event Descriptor (HED) Tags
   leads:
     - name: Chris Gorgolewski
   content:
     - raw
+  google_doc_created: 2017-07
+  pull_request_created: # only ever existed as a google doc ?
+  pull_request_merged: 2018-04
 
 - number: "008"
   title: Magnetoencephalography (MEG)
   leads:
     - name: Guiomar Niso
   content:
     - raw
+  google_doc_created: 2016-03
+  pull_request_created: # only ever existed as a google doc ?
+  pull_request_merged: 2018-04
 
 - number: "009"
   title: Positron Emission Tomography (PET)
   leads:
     - name: Melanie Ganz
   content:
     - raw
+  google_doc_created:
```

Comment:
Already existed as of March 2018: https://groups.google.com/g/bids-discussion/c/lyuG4Pv_5UI/m/LxFDEmeNBwAJ

Possibly existed back in Oct 2017: https://groups.google.com/g/bids-discussion/c/9uJJ0lgnt2g/m/ChtWAQNUAgAJ

@melanieganz or @mnoergaard might be able to look through the version history to determine when it was started. I don't have that access.

Reply:
got the date from @melanieganz